src/main/java/org/terasology/web/resources/OnlinePlayersResource.java

```diff
@@ -52,11 +53,9 @@ public void onDisconnected(DisconnectedEvent event, EntityRef entityRef) {
     }
 
     @Override
-    public List<String> read(Client requestingClient) {
-        List<String> result = new ArrayList<>();
-        for (Client client: networkSystem.getPlayers()) {
-            result.add(client.getName());
-        }
-        return result;
+    public List<OnlinePlayerMetadata> read(Client requestingClient) {
+        return StreamSupport.stream(networkSystem.getPlayers().spliterator(), true)
```

Comment:
Just checked, `getPlayers()` returns an `Iterable<Client>` ([source](https://github.com/MovingBlocks/Terasology/blob/develop/engine/src/main/java/org/terasology/network/NetworkSystem.java#L50-L50)) which hasn't the `.stream()` method. So the only way I found to get a stream was using StreamSupport (I read it [here](https://stackoverflow.com/a/23936723)).

An alternative could be a plain old foreach which iterates over the `getPlayers()` results and adds the metadata entries to another collection, than return said collection.